scripts/screeps_tencent_batch_rl_runner.py

```
@@ -2064,6 +2086,202 @@ def verified_remote_policy_update_fields(
+                f"remote candidate scorecard artifact was not collected: "
+                f"{rel_scorecard_artifact_path.as_posix()}"
+            )
+    elif status == "blocked":
+        classification = raw.get("classification")
+        materialization_failed = classification == "candidate_scorecard_materialization_failed"
+        top_level_runtime_injected = (
+            runtime_parameter_injection is not None
+            and runtime_parameter_injection.get("runtimeParameterInjection") is True
+        )
+        if runtime_injected and not materialization_failed:
+            raise BatchRunError("remote candidateScorecard blocked status requires runtimeParameterInjection=false")
```

Ruff (0.15.13)

[warning] Avoid specifying long messages outside the exception class

(TRY003)